test/completion/comprehensions.py

```diff
@@ -50,6 +50,13 @@
 ##? str()
 {a-1:b for a,b in {1:'a', 3:1.0}.items()}[0]
 
+# with a set literal
+#? int()
+[a for a in {1, 2, 3}][0]
+
+##? int()
```

Comment:
It was not intentional, my mistake. I updated the test.
A few additional tests are disabled intentionally, because these tests needs full support of dict/set generator class.
I am happy to help to add this generator class, but really need to study the source further.